lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -120,7 +120,7 @@ function Test-ProjectPackage {
   param ($Path, $Algorithm = 'SHA256', $Hash)
   if (!$env:Valid_ProjectPackage){
     Write-Verbose "Testing the $Algorithm hash for $path."
-    $ActualHash = if (Is-FIPS) {
+    $ActualHash = if ($(Is-FIPS) -ge 1 -or $(Get-PowershellVersion) -le 3) {
```

Comment:
is there a reason not to use `Custom-GetFileHash` for all cases to simplify this?